test/acceptance/test/add_tests.go

```diff
@@ -277,7 +277,7 @@ var _ = Describe("Weave GitOps Add Tests", func() {
 			createGitRepoBranch(repoAbsolutePath, branchName)
 		})
 
-		By("And I run wego add command with specified branch, namespace, url", func() {
+		By("And I run wego add command with specified branch, namespace, url, deplyment-type, private-key", func() {
```

Comment:
I don't see the private-key flag used in this test?

Reply:
oh - i wanted to add test for the flag on a different PR, left the comment accidentally. Will get rid of it